operator/gitops/argocd/tekton-results/deployment.yaml

```diff
@@ -38,21 +38,28 @@ spec:
                   name: tekton-results-database
             - name: DB_PROTOCOL
               value: tcp
-            - name: DB_ADDR
+            - name: DB_HOST
               value: tekton-results-database-service.tekton-results.svc.cluster.local
             - name: DB_NAME
               value: tekton_results
-          image: quay.io/redhat-appstudio/tekton-results-api@sha256:dca434dccd576e7a186098779f678422f10ac16b395c67944ffa50633fc3dd2e
+          image: quay.io/redhat-appstudio/tekton-results-api@sha256:56471d0c0582e8c4e4c109ef615b66e94a7fba258ab5960e36adce71d5a6e7f1
           name: api
           volumeMounts:
             - mountPath: /etc/tls
               name: tls
               readOnly: true
+            - name: config
+              mountPath: /config/env
```

Comment:
Can you please clarify how these mount paths are determined? I am a bit confused. If I understood correctly it can be any of the path defined in api/main.go. 

https://github.com/openshift-pipelines/tektoncd-results/blob/b967dde4d941415f239e89403a7d65c61e9f1b2c/cmd/api/main.go#L51

Reply:
Since the api binary is run from /usr/local/bin it resolves the path from the root path. 
adding the `.` in the path works the same way so Im happy to update the deployment if you would prefer it that way?

Reply:
@avinal @CathalOConnorRH Let me know when this conversation is resolved, and I'll merge the PR.

Reply:
Yes, we discussed it and this is clear to me now. This looks fine.